src/table/utils/handle-scroll.js

```diff
@@ -1,11 +1,25 @@
-export const handleScroll = (evt, tableContainerRef) => {
+export const handleHorizontalScroll = (evt, rtl, memoedContainer) => {
   evt.stopPropagation();
-  let { scrollLeft } = tableContainerRef.current;
-  const newScrollLeft = scrollLeft + evt.deltaX;
-  const max = tableContainerRef.current.scrollWidth - tableContainerRef.current.offsetWidth;
-  if (max > 0 && (newScrollLeft < 0 || newScrollLeft > max)) {
+  // scrollWidth is the width of an element's content, including content not visible on the screen due to overflow.
+  // offsetWidth is the element's CSS width, including any borders, padding, and vertical scrollbars
+  // scrollLeft is the number of pixels scrolled from its left edge
+  let { scrollLeft } = memoedContainer;
```

Comment:
You can destruct `scrollWidth` and `offsetWidth` too if you want

Reply:
yes, I want that too. But eslint complaints that scrollWidth and offsetWidth should be const not let.

Reply:
ah ok, then leave it